bin/install_dependencies.ps1

```diff
@@ -34,6 +36,11 @@ if (Test-Path($ChocolateyProfile)) {
 }
 refreshenv
 
+Write-Output " ***** Seting up MSYS ******* "
```

Comment:
Small typo, should be `Setting`

Reply:
Oops, thanks for noticing, Ronny. Will fix it.

Reply:
Fixed it